Action.c

```diff
@@ -216,6 +216,16 @@ static Htop_Reaction actionToggleKernelThreads(State* st) {
 
 static Htop_Reaction actionToggleUserlandThreads(State* st) {
    st->settings->hideUserlandThreads = !st->settings->hideUserlandThreads;
+
+   if (st->settings->hideUserlandThreads) {
+      /* Do not show userland threads as tombed */
+      ProcessList_noTombedUserlandThreads(st->pl);
+   } else {
+      /* Do not show userland threads as new */
+      ProcessList_scan(st->pl, false);
+      ProcessList_noNewUserlandThreads(st->pl);
```

Comment:
This discards legitimately new userland threads.

Reply:
With this patch userland threads are not parsed at all when disabled. So the are legitimately new on first scan after toggle-ling the setting.
It will only affect userland threads started **exactly** at the same time as the setting is switched, as there is one scan before (line 225), and one scan directly after https://github.com/htop-dev/htop/blob/be568b1153206a067fd09c0f7d2dcea3d7dbaa68/ScreenManager.c#L115 (cause we return `HTOP_RECALCULATE`)

Reply:
What about tracking their PIDs (with `show==false`), but not filling in any other details while hiding userland threads?

Reply:
moved the thread skipping to #334